wrappers/python/third_party/pybind11/MANIFEST.in

```diff
@@ -1,2 +1,2 @@
-include include/pybind11/*.h
+recursive-include include/pybind11 *.h
```

Comment:
The pybind source code differ significantly. Do we need to update the installation instruction as to which pybind version is required?

Reply:
> #### Changes:
> - upgrade to pybind11 v2.2

I don't think we have the version of pybind any where in our docs